sonic_platform_base/sonic_xcvr/api/credo/aec_800g.py

```diff
@@ -0,0 +1,17 @@
+"""
+    aec_800g.py
+
+    Implementation of Vendor specific in addition to the CMIS specification.
+"""
+
+from ...fields import consts
+import logging
+from ..public.cmis import CmisApi
+import time
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
```

Comment:
@AnoopKamath seems like we are not using these?

Reply:
It was added while debugging. Removed now